.github/workflows/go_test.yaml

```diff
@@ -0,0 +1,62 @@
+name: "Go Test and Coverage"
+
+on:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: ./go.mod
+
+      - name: Install dependencies
+        run: |
+          go install github.com/boumenot/gocover-cobertura@latest
+
+      - name: Set up gotestfmt
+        uses: GoTestTools/gotestfmt-action@v2
+        with:
+          version: v2.5.0
+
+      - name: Run tests with coverage
+        run: |
+          go test -race -cover -coverpkg=./... -gcflags="all=-N -l" -v -coverprofile=cover.out -json -v $(go list ./... | grep -Ev '/internal/server/infrastructure/mysql/db|log_service/pkg/logger|log_service/pkg/rabbitmq|log_service/internal/rabbitmq') 2>&1 | tee /tmp/gotest.log | gotestfmt
```

Comment:
Why does it execute grep -Ev?

Reply:
It used in the old implementation to group directories that were not tested. But for now, I deleted it as I use octocov now.